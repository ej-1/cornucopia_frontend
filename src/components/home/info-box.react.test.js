import React from "react";
import InfoBox from "../home/info-box";
import renderer from "react-test-renderer";

test("InfoBox gets rendered", () => {
  let infoBoxProps = {
    header: "Run trade simulations",
    text:
      "With CornucopiaTrader you simulate trading based on technical \n \
            indicators on historical exchange data from Binance.",
    buttonLink: "/simulate",
    buttonStyle: "primary",
    buttonText: "Simulate now"
  };

  const component = renderer.create(<InfoBox {...infoBoxProps} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
