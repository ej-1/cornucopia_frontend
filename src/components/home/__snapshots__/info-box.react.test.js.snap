// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBox gets rendered 1`] = `
<div
  className="col-md-6 col-xs-12"
>
  <div
    className="page-header"
  >
    <h1>
      <span>
        Run trade simulations
        <br />
      </span>
    </h1>
  </div>
  <p>
    <span>
      With CornucopiaTrader you simulate trading based on technical 
      <br />
    </span>
    <span>
                   indicators on historical exchange data from Binance.
      <br />
    </span>
  </p>
  <a
    className="btn btn-primary"
    href="/simulate"
    onClick={[Function]}
    onKeyDown={[Function]}
  >
    Simulate now
  </a>
</div>
`;
